src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/payments/PaymentsPublisher.java

```diff
@@ -34,24 +36,30 @@ public PaymentsPublisher(
 
     @Override
     public void send(CreatePaymentsCommand cmd) {
-        try {
-            String messageBody = objectMapper.writeValueAsString(cmd);
+        send(cmd, Labels.CREATE);
+    }
 
+    @Override
+    public void send(UpdatePaymentsCommand cmd) {
+        send(cmd, Labels.UPDATE);
+    }
+
```

Comment:
IMHO should be better to fetch label from the command itself. There should be some interface defining getLabel() method implemented in each command class. This will end up in having just one send() method.